components/dropdown/dropdown-content-mixin.js

```diff
@@ -477,8 +483,9 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 
 			const maxHeight = Math.floor((this.openedAbove ? spaceAround.above : spaceAround.below) - headerFooterHeight);
```

Comment:
The variable names have becomes somewhat confusing.  Is my understanding correct that this `maxHeight` is effectively the maximum possible height for the **content**, as constrained by the viewport?  Whereas, `this.maxHeight` (referenced in the code changes below) is the maximum height for the entire dropdown including the header and footer that's desired by the consumer.  And, ultimately the maximum height that the component uses is the smaller of the two, factoring in the header and footer?

Reply:
I've updated the variable names into something that is hopefully easier to read and understand. 🤔 feedback welcome.